index.html

```diff
@@ -244,8 +254,17 @@ <h3>
                     </li>
                 </ul>
             </form>
-            <p>ghauskhan1616@gmail.com</p>
+
         </div>
+    </section>
+    <p class="contactEmail">ghauskhan1616@gmail.com</p>
+    <div class="contactBackground">
+        <img class="illustration1" src="Assets/illustration-1-contact-form-desktop.png" alt="Illustration1">
+        <img class="illustration2" src="Assets/illustration-2-contact-form-desktop.png" alt="Illustration2">
+        <img class="illustration3" src="Assets/contact form-illustratoin-3-desktop.png" alt="Illustration3">
+
+    </div>
+    <section class="bottom_bar">
         <div class="socialIcons">
```

Comment:
kindly implement these changes for the `contact me ` section of your project :

- [ ] Please kindly reduce the spacing in the text on the left of this section
- [ ] I think you should increase the padding and font size of the button
- [ ] Also kindly add the horizontal line close to the social icon as it is in the design
- [ ] I'd suggest you please reduce the spacing close to the social icons as well 😄 

your design :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/5d50374d-2c3b-42a1-bb8a-6b19f4d89db6)

Design given :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/b8bf0ecb-4f9e-4276-822a-bee83f496def)